p2p/switch.go

```diff
@@ -346,17 +353,20 @@ func (sw *Switch) DialPeersAsync(addrBook AddrBook, peers []string, persistent b
 	for _, err := range errs {
 		sw.Logger.Error("Error in peer's address", "err", err)
 	}
+	if len(errs) > 0 {
+		return errors.New("Errors in peer addresses (see errors above)")
```

Comment:
hmm, this means if there are lookup errors or typos in the persistent_peers, tendermint will fail to start ? didn't some one ask for us not to do this previously ?

Reply:
right right